qemu/tests/virtio_serial_file_transfer.py

```diff
@@ -161,7 +161,35 @@ def transfer_data(params, vm, host_file_name=None, guest_file_name=None,
     :param sender: Who send the data file
     :return: True if pass, False and error message if check fail
     """
+    def kill_pid():
```

Comment:
Why do we need this? If there is process not quit as expected, I suppose we should kill it at the end of the case.

Reply:
virtio_serial_file_transfer is a background case, guest maybe reboot during this case, and after reboot we need to run virtio_serial_file_transfer case again.
If host process still exists, two dup host thread will be open, data transfer will fail.
So need to clean it before run again.

Reply:
So your suggestion is to move it after _transfer_data?
I can try with that

Reply:
> virtio_serial_file_transfer is a background case, guest maybe reboot during this case, and after reboot we need to run virtio_serial_file_transfer case again.
> If host process still exists, two dup host thread will be open, data transfer will fail.
> So need to clean it before run again.

No, don't use the whole test as a background test, it's hard to make sure the process correct, please write new case utilize functions instead of use `utils_test.run_virt_sub_test`.